src/mds/MDCache.cc

```diff
@@ -13701,6 +13705,13 @@ void MDCache::dispatch_quiesce_inode(const MDRequestRef& mdr)
     } else {
       // replica will follow suite and move to LOCK_LOCK state
       // as a result of the auth taking the above locks.
+
+      /* kick cap issuance without waiting for auth */
+      if (quiesce_replica_recall && !in->is_auth()) {
```

Comment:
`&& !in->is_auth()` is redundant, we are already in the `else` branch of the inverted test.

Reply:
right, this was just the state after a trivial rebase, see now